public/plugin.tsx

```diff
@@ -46,23 +51,28 @@ export class AssistantPlugin
     const contentRenderers: Record<string, ContentRenderer> = {};
     const actionExecutors: Record<string, ActionExecutor> = {};
     const assistantActions: AssistantActions = {} as AssistantActions;
-    const getAccount = async () => {
-      return await core.http.get<{
-        data: { roles: string[]; user_name: string; user_requested_tenant: string | null };
-      }>('/api/v1/configuration/account');
-    };
-    const assistantEnabled = (() => {
-      let enabled: boolean;
+    /**
+     * Returns {@link UserAccountResponse}. Provides default roles and user
+     * name if security plugin call fails.
+     */
+    const getAccount: () => Promise<UserAccountResponse> = (() => {
+      let account: UserAccountResponse;
       return async () => {
-        if (enabled === undefined) {
-          const account = await getAccount();
-          enabled = account.data.roles.some((role) =>
-            ['all_access', 'assistant_user'].includes(role)
-          );
+        if (setupDeps.securityDashboards === undefined)
+          return { data: { roles: ['all_access'], user_name: 'dashboards_user' } };
+        if (account === undefined) {
```

Comment:
The `account` is always be undefined here, shall we remove this check?

Reply:
not on the second `getAccount()` call